src/Microsoft.Health.Fhir.Liquid.Converter.UnitTests/Filters/DateFiltersTests.cs

```diff
@@ -306,6 +316,14 @@ public void GivenAnInvalidTimeZoneHandling_WhenFormatAsDateTime_ExceptionShouldB
             Assert.Equal(FhirConverterErrorCode.InvalidTimeZoneHandling, exception.FhirConverterErrorCode);
         }
 
+        [Theory]
+        [MemberData(nameof(GetValidDataForFhirToHl7v2Conversion))]
+        public void GivenAValidData_WhenFormatAsFhirDateTime_ConvertedHl7v2DateTimeShouldBeReturned(string input, string expectedDateTime)
+        {
+            var result = Filters.FormatAsHl7v2DateTime(input);
+            Assert.Equal(expectedDateTime, result);
+        }
```

Comment:
minor: could you also add the invalid test cases for expected error code.

Reply:
added